src/scenes/Root/Sidebar/sidebar.theme.ts

```diff
@@ -0,0 +1,27 @@
+import { createTheme } from '../../../theme';
+
```

Comment:
When do we create a separate theme and override styles vs just overriding with classes like before? I'm not really partial to either way but wanted to know if there was some cutoff for when we should do one vs the other.

Reply:
We talked on Zoom, but to summarize here:
- Basically never create a separate theme lol
- classes vs changing app theme is a judgement call. It's probably better to start with classes and then if we see a trend we can move it to theme.
  Layout should probably mostly just be in classes.
  Colors, fonts, etc should mostly come from theme.